openlibrary/plugins/worksearch/code.py

```diff
@@ -987,7 +987,7 @@ def random_author_search(limit=10):
         # The template still expects the key to be in the old format
         doc['key'] = doc['key'].split("/")[-1]
 
-    return json.dumps(search_results['response'])
+    return search_results['response']
```

Comment:
Can you update this function's documentation?  Replace "Returns a JSON string" with "Returns a dict" on line 967.

Reply:
Done :)